public/js/common.js

```diff
@@ -1,30 +1,30 @@
-
-window.schuldhulp = window.schuldhulp || {};
-
-window.schuldhulp._ = window.schuldhulp._ || {};
-
-window.schuldhulp._.matches = function(el, selector) {
-    return (el.matches || el.matchesSelector || el.msMatchesSelector || el.mozMatchesSelector || el.webkitMatchesSelector || el.oMatchesSelector).call(el, selector);
-};
-
-window.schuldhulp._.findParent = function(elm, find) {
-    if (window.schuldhulp._.matches(elm, find)) {
-        return elm;
-    }
-    if (elm.parentNode === null) {
-        return null;
-    }
-    return window.schuldhulp._.findParent(elm.parentNode, find);
-};
-
-window.schuldhulp._.datePicker = function (elm) {
-    rome(elm, {
-        'inputFormat': 'DD-MM-YYYY',
-        'time': false,
-        'moment': {
-            'locale': 'nl'
-        }
-    });  
-};
-
+
+window.schuldhulp = window.schuldhulp || {};
+
+window.schuldhulp._ = window.schuldhulp._ || {};
+
+window.schuldhulp._.matches = function(el, selector) {
+    return (el.matches || el.matchesSelector || el.msMatchesSelector || el.mozMatchesSelector || el.webkitMatchesSelector || el.oMatchesSelector).call(el, selector);
+};
+
+window.schuldhulp._.findParent = function(elm, find) {
+    if (window.schuldhulp._.matches(elm, find)) {
+        return elm;
+    }
+    if (elm.parentNode === null) {
+        return null;
+    }
+    return window.schuldhulp._.findParent(elm.parentNode, find);
+};
+
+window.schuldhulp._.datePicker = function (elm) {
+    rome(elm, {
+        'inputFormat': 'DD-MM-YYYY',
+        'time': false,
+        'moment': {
+            'locale': 'nl'
+        }
+    });  
+};
+
```

Comment:
Hier zijn de codewijzigingen niet leesbaar. Als er geen codewijzigingen zijn, moet dit stuk er uit of anders alleen de diffs tonen.

Reply:
Zie mijn [algemene reactie](https://github.com/Amsterdam/schulddossier/pull/277#issuecomment-1097824167), er is gelukkig een mogelijkheid in GitHub aanwezig om deze whitespace change weg te filteren.